api/src/main/native/djl/utils.h

```diff
@@ -100,9 +111,22 @@ inline std::vector<std::string> GetVecFromJStringArray(JNIEnv* env, jobjectArray
 // String[]
 inline jobjectArray GetStringArrayFromVec(JNIEnv* env, const std::vector <std::string> &vec) {
   jobjectArray array = env->NewObjectArray(vec.size(), env->FindClass("Ljava/lang/String;"), nullptr);
+
```

Comment:
you can put the comment here saying cache the java reflection method to get better perf.